### PR TITLE
[PR-16730] SPM distribution documentation for iOS SDK and polishing

### PR DESCRIPTION
**Pull request by dgtv**
Documentation update for https://github.com/talkable/ios-sdk/pull/16

## Demo

https://deploy-preview-275--talkable-docs.netlify.app/

## Related Stories

[](https://talkable.atlassian.net/browse/PR-16730)

### CODE REVIEW

File: source/ios_sdk/getting_started.rst [hunk 1]
```diff
@@ -67,44 +90,38 @@ Configuration
         <dict>
           <key>CFBundleURLSchemes</key>
           <array>
-            <string>tkbl-<your-site-slug></string>
+            <string>tkbl-<your-site-id></string>
```
Comment (zhuravel) [inline source/ios_sdk/getting_started.rst:93]: @dgtv There are more places where slug is still slug:

**Review by zhuravel — CHANGES_REQUESTED** [review]

File: source/ios_sdk/getting_started.rst (hunk 1 above)
Comment (dgtv) [reply source/ios_sdk/getting_started.rst:93]: Ok, I'll remove changes related to `site_slug` from this PR and will introduce them along this task  https://talkable.atlassian.net/browse/PR-18436 and PR https://github.com/talkable/talkable/pull/8512 as it also requires such changes anyway.

File: source/ios_sdk/getting_started.rst [hunk 2]
```diff
@@ -78,33 +104,27 @@ Configuration
 
       <key>LSApplicationQueriesSchemes</key>
       <array>
-        <string>tkbl-your-site-slug</string>
+        <string>tkbl-<your-site-slug></string>
```
Comment (zhuravel) [inline source/ios_sdk/getting_started.rst:107]: Let’s use some other placeholder format that does not look like XML tag in an XML snippet and does not make it seem invalid (no closing tag).
```suggestion
        <string>tkbl-YOUR-SITE-SLUG</string>
```

File: source/ios_sdk/integration/post_purchase.rst [hunk 3]
```diff
@@ -6,13 +6,13 @@ Post Purchase Campaign
 
 Now that we know what a Standalone campaign is, let’s take a look at one more type of Talkable campaign — Post Purchase. It is used to convert your existing customers into Advocates after they make a purchase.  It has the same flow as a basic Standalone campaign except this is gated by a purchase and shown to a customer immediately after.
 
-Post Purchase campaign usually looks like a pop up right after a user made a purchase. This campaign initializes on the order confirmation page and captures order details parameters. You need to pass order data that includes your customer’s email address, order id, subtotal, coupon code used at checkout in order to close a referral loop. This allows Talkable to detect and close the loop on a referral.
+Post Purchase campaign usually looks like a pop up right after a user made a purchase. This campaign is initialized on the order confirmation page and captures order with it's details. You need to pass order data to Talkable that includes your customer’s email address, order number, subtotal, coupon code used at checkout to allow Talkable detect and close a referral loop.
```
Comment (zhuravel) [inline source/ios_sdk/integration/post_purchase.rst:9]: https://boldomatic.com/p/xj2qSA/you-re-you-fucking-are-your-shows-fucking-possession-they-re-they-fucking-are-th

```suggestion
Post Purchase campaign usually looks like a pop up right after a user made a purchase. This campaign is initialized on the order confirmation page and captures order with its details. You need to pass order data to Talkable that includes your customer’s email address, order number, subtotal, coupon code used at checkout to allow Talkable detect and close a referral loop.
```

File: source/ios_sdk/integration/standalone.rst [hunk 4]
```diff
@@ -12,28 +12,28 @@ Once you’ve got a Standalone campaign set up inside Talkable you can integrate
...
+  @import TalkableSDK;
   ...
   NSDictionary* params = @{
     TKBLAffiliateMemberKey: @{
       TKBLAffiliateMemberEmailKey: @"advocate@example.com",
       TKBLAffiliateMemberFirstNameKey: @"John",
       TKBLAffiliateMemberLastNameKey: @"Smith",
       TKBLAffiliateMemberPersonCustomPropertiesKey: @{
-        @"eye_color": @"brown"
+        @"eye_color": @"green"
```
Comment (zhuravel) [inline source/ios_sdk/integration/standalone.rst:36]: Just wondering – what made you change this?

**Review by zhuravel — APPROVED** [review]

File: source/ios_sdk/integration/standalone.rst (hunk 4 above)
Comment (dgtv) [reply source/ios_sdk/integration/standalone.rst:36]: Just read text above =)
> For example, below we are creating and passing a custom key value pair of *eye color* = *green*. Any data passed through here will be accessible on a campaign level for segmentation or other logic.

**Review by zhuravel — APPROVED** [review]

File: source/ios_sdk/api_integration.rst [hunk 5]
```diff
@@ -87,7 +86,7 @@ via |api_v2_share_via_social_channel|.
                                 withContent:content
                                    delegate:delegate];
 
-   ...
+   //...
```
Comment (zhuravel) [inline source/ios_sdk/api_integration.rst:89]:
```suggestion
   // ...
```

**Review by zhuravel — APPROVED** [review]

**Pull request merged**